examples/apps/branding/app.py

```diff
@@ -209,13 +210,12 @@ def get_continous_plot(dark_theme):
             docs_link,
             sizing_mode="stretch_both",
             margin=(10, 10),
-            styles={"border": "1px solid gray"},
         ),
     ],
     sidebar_width=425,
-    main=[pmui.Container("<h2>Buttons</h2>", example_buttons, "<h2>Plots</h2>", categorical_plot, continous_plot, drawer)],
+    main=[pmui.Container("## Buttons", example_buttons, "## Plots", categorical_plot, continous_plot, drawer)],
```

Comment:
The reason why I use HTML is that I want to avoid the anchor links. They make things jump. See https://github.com/holoviz/panel/issues/7878.